Emby/dist/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+  "author": {
+    "name": "alexbcberio",
+    "id": "202915432175239169"
+  },
+  "url": "emby.media",
+  "description": {
+    "en": "Your personal media on any device. Bringing all of your home videos, music, and photos together into one place has never been easier. Your personal Emby Server automatically converts and streams your media on-the-fly to play on any device."
+  },
+  "service": "Emby",
+  "version": "1.0",
+  "logo": "https://imgur.com/download/mTHBDPm",
+  "thumbnail": "https://imgur.com/download/a80aJAx",
```

Comment:
These must be `i.imgur` links.

Reply:
Update in commit [e4e9b9f78234833a1a64cb3093f3ee71fc0cc253](https://github.com/alexbcberio/Presences/commit/e4e9b9f78234833a1a64cb3093f3ee71fc0cc253)